### ISSUE TITLE
2.0.8 import broken (Python3) - 'str' object has no attribute 'mayReturnEmpty'

### ISSUE THREAD

**Issue by wesinator** [fhightower/ioc-finder#58]
```python traceback
>>> import ioc_finder
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/ioc_finder/__init__.py", line 4, in <module>
    from .ioc_finder import (
  File "/usr/local/lib/python3.7/site-packages/ioc_finder/ioc_finder.py", line 14, in <module>
    import ioc_grammars
  File "/usr/local/lib/python3.7/site-packages/ioc_finder/ioc_grammars.py", line 321, in <module>
    CaselessLiteral('tlp') + Or([':', '-', ' ', Empty()]).setParseAction(lambda x: ':') + tlp_colors
  File "/usr/local/lib/python3.7/site-packages/pyparsing.py", line 3769, in __init__
    self.mayReturnEmpty = any(e.mayReturnEmpty for e in self.exprs)
  File "/usr/local/lib/python3.7/site-packages/pyparsing.py", line 3769, in <genexpr>
    self.mayReturnEmpty = any(e.mayReturnEmpty for e in self.exprs)
AttributeError: 'str' object has no attribute 'mayReturnEmpty'
```

**Comment by wesinator** [comment]
workaround for posterity : 
```sh
pip install git+https://github.com/fhightower/ioc-finder.git@6f9a324ea7d34aebc97a5dde9990c7c614a99fe0
```

**Comment by fhightower** [comment]
The cause of this problem is described here: https://sourceforge.net/p/pyparsing/bugs/108/ .

TL;DR: pyparsing gets confused if given an `Or` statement with both strings and expressions in it (as is done here: https://github.com/fhightower/ioc-finder/blob/a2ce96cf8641f697d5877341b0a4b5866996aeea/ioc_finder/ioc_grammars.py#L321)

**Comment by fhightower** [comment]
Thanks for reporting this Wes! I've pushed an update to fix this problem (and have released version 2.0.9 to pypi to fix this).

**Comment by wesinator** [comment]
thanks - will tests catch future import issues like this ?

**Comment by fhightower** [comment]
I ran the tests on a mac and it didn't throw this error so I can't make any guarantees, but I will try to run tests in different OSes to prevent further problems of this kind.
